views/getcoverage.inc.php

```diff
@@ -1,7 +1,13 @@
 <?php
 namespace Langchecker;
 
-include __DIR__ . '/../config/adu.inc.php';
+// Real data is in adi.inc.php, not under VCS
+if (is_file(__DIR__ . '/../config/adi.inc.php')) {
+    include __DIR__ . '/../config/adi.inc.php';
+} else {
+    // Fake data to not break the app outside of production
+    include __DIR__ . '/../config/fake_adi.inc.php';
+}
 
```

Comment:
Uhm... Do we actually need this? We should already have access to ´´´sources.php´´´